community/kernel/src/main/java/org/neo4j/storageengine/api/txstate/NodeState.java

```diff
@@ -59,5 +60,9 @@ void visitLabelChanges( long nodeId, Set<Integer> added, Set<Integer> removed )
 
     PrimitiveLongIterator getAddedRelationships( Direction direction, int[] relTypes );
 
+    PrimitiveLongIterator getAddedRawRelationships();
```

Comment:
What is "raw relationships"? Please take the opportunity to describe that in javadocs.